bin/pycbc_make_skymap

```diff
@@ -465,4 +487,4 @@ if __name__ == '__main__':
          test_event=not opt.enable_production_gracedb_upload,
          custom_frame_files=opt.custom_frame_file,
          approximant=opt.approximant, detector_state=opt.detector_state,
-         veto_definer=opt.veto_definer)
+         veto_definer=opt.veto_definer, injection_file=opt.injection_file)
```

Comment:
You also need to pass the values given to `--fake-strain` and `--fake-strain-seed`.